test/basic.js

```diff
@@ -29,14 +29,19 @@ rimraf.sync(dir)
 mkdirp.sync(dir)
 
 const keys = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
+const config = {
+  keys,
+  path: dir,
+  db2: {
+    flushDebounce: 10,
+    writeTimeout: 10
+  }
+}
```

Comment:
this config change took the run time of this file from 14s down to 4s .....

Reply:
<future> someone should make a test/helpers/testbot.js so it's easy to apply this win across the whole repo....